src/main/java/com/minecolonies/commands/ColonyInfo.java

```diff
@@ -0,0 +1,146 @@
+        final boolean found;
+
+        if (args.length != 0)
+        {
+            try
+            {
+                colonyId = Integer.parseInt(args[0]);
+            }
+            catch (NumberFormatException e)
+            {
+                if(!args[0].isEmpty())
+                {
+                    mayorID = getUUIDFromString(sender, args);
+                }
+            }
+        }
+
+        final IColony tempColony;
+        if(colonyId == -1)
+        {
+            tempColony = ColonyManager.getIColonyByOwner(sender.getEntityWorld(), mayorID);
+        }
+        else
+        {
+            tempColony = ColonyManager.getColony(colonyId);
+        }
+
+        if(tempColony == null)
+        {
+            sender.addChatMessage(new TextComponentString(String.format(NO_COLONY_FOUND_MESSAGE_ID, colonyId)));
+            return;
+        }
+
+        Colony colony = ColonyManager.getColony(sender.getEntityWorld(), tempColony.getCenter());
+        if (colony == null)
+        {
+            sender.addChatMessage(new TextComponentString(String.format(NO_COLONY_FOUND_MESSAGE_ID, colonyId)));
+            return;
+        }
+
+        colonyId = colony.getID();
+        if (colonyId == -1)
+        {
+            sender.addChatMessage(new TextComponentString(String.format(NO_COLONY_FOUND_MESSAGE, args[0])));
+            return;
+        }
+        
+        final BlockPos position = colony.getCenter();
+        sender.addChatMessage(new TextComponentString(ID_TEXT + colony.getID() + NAME_TEXT + colony.getName()));
+        final String mayor = colony.getPermissions().getOwnerName();
+        sender.addChatMessage(new TextComponentString(MAYOR_TEXT + mayor));
+        sender.addChatMessage(new TextComponentString(CITIZENS + colony.getCitizens().size() + "/" + colony.getMaxCitizens()));
+        sender.addChatMessage(new TextComponentString(COORDINATES_TEXT + String.format(COORDINATES_XYZ, position.getX(), position.getY(), position.getZ())));
+    }
+
+    private static UUID getUUIDFromString(@NotNull final ICommandSender sender, @NotNull final String... args)
+    {
+        final MinecraftServer tempServer = sender.getEntityWorld().getMinecraftServer();
+        if(tempServer != null)
+        {
+            final GameProfile profile = tempServer.getPlayerProfileCache().getGameProfileForUsername(args[0]);
+            if (profile != null)
+            {
+                return profile.getId();
+            }
+        }
+        return null;
+    }
+
+    @NotNull
+    @Override
+    public List<String> getTabCompletionOptions(
+                                                 @NotNull final MinecraftServer server,
+                                                 @NotNull final ICommandSender sender,
+                                                 @NotNull final String[] args,
+                                                 @Nullable final BlockPos pos)
+    {
+        return new ArrayList<>();
```

Comment:
We should save this as a constant, so we don't have to make a new list. Also there might just be a Collections.emptyList()